cognite/powerops/custom_modules/power_model_v1/data_models/views/function_inputs/BenchmarkingShopRunInputDayAhead.view.yaml

```diff
@@ -0,0 +1,65 @@
+space: "{{power_ops_models}}"
+externalId: BenchmarkingShopRunInputDayAhead
+name: BenchmarkingShopRunInputDayAhead
+description: Class for inputs to shop runs used for day ahead benchmarking
+filter:
+  and:
+    - hasData:
+        - type: container
+          space: "{{power_ops_models}}"
+          externalId: FunctionData
+    - equals:
+        property:
+          - node
+          - space
+        value: "{{power_ops_instance_space}}"
+    - equals:
+        property:
+          - node
+          - type
+        value:
+          externalId: BenchmarkingShopRunInputDayAhead
+          space: "{{power_ops_type_space}}"
+implements:
+  - space: "{{power_ops_models}}"
+    externalId: ShopTriggerInput
+    version: "{{version}}"
+    type: view
+version: "{{version}}"
+properties:
+  case:
+    container:
+      space: "{{power_ops_models}}"
+      externalId: FunctionData
+      type: container
+    containerPropertyIdentifier: direct1
+    name: case
+    description: "The SHOP case (with all details like model, scenario, and time series)"
+    source:
+      space: "{{power_ops_models}}"
+      externalId: ShopCase
+      version: "{{version}}"
+      type: view
```

Comment:
**question:** case is already a property on `ShopTriggerInput` so you're just overriding the definition of the property here with no changes except for the _description_?